src/v1/controllers/news.controller.js

```diff
@@ -0,0 +1,54 @@
+const catchAsync = require("../../utils/catchAsync");
+const {PreferenceService} = require("../services");
+require('dotenv').config()
+const axios = require('axios')
+
+let everythingEndpoint = 'https://newsapi.org/v2/everything'
+let topHeadlinesEndpoint = 'https://newsapi.org/v2/top-headlines'
+
+const getNews = catchAsync(async (req, res) => {
+    let {user} = req
+    let preferences = await PreferenceService.getUserPreferences(user.id)
+    console.log(preferences)
+    let userKeywords = preferences || []
+    console.log("userKeywords")
```

Comment:
This looks more like a debug logs. You should also not log too much unnecessary logs, as the storage and querying comes with a cost... :)